src/modules/ens/sagas.ts

```diff
@@ -497,6 +501,52 @@ export function* ensSaga(builderClient: BuilderClient, ensApi: ENSApi, worldsAPI
     }
   }
 
+  function* handleFetchContributableNamesRequest() {
+    try {
+      let names: ContributableDomain[] = yield call([ensApi, worldsAPIContent.fetchContributableDomains])
+
+      const bannedNames: string[] = yield call([lists, 'fetchBannedNames'])
```

Comment:
The functionality to get the banned names seems to be repeated in this file, would you mind abstracting it? Be aware that you'll need to mock the `fetchWorldDeploymentsRequest` to prevent this saga from executing it.